includes/agentic-commerce/class-wc-stripe-agentic-commerce-product-mapper.php

```
@@ -821,6 +844,59 @@ protected function get_shipping(): ?string {
+					$has_flat_rate = true;
+					break;
+				}
+			}
+
+			if ( ! $has_flat_rate ) {
+				$zones_without_flat_rate[] = $zone_name;
+			}
+		}
+
+		return [ 'zones_without_flat_rate' => array_values( array_unique( $zones_without_flat_rate ) ) ];
+	}
```

PHPMD (2.15.0)

[error] The method get_shipping_diagnostics is not named in camelCase. (undefined)

(CamelCaseMethodName)